cockatrice/CMakeLists.txt

```diff
@@ -80,6 +81,7 @@ SET(cockatrice_SOURCES
     src/tab_userlists.cpp 
     src/tab_deck_editor.cpp
     src/tab_logs.cpp
+    src/tip_of_the_day.cpp
```

Comment:
Can we put the two tip lines together in the CMake file? Just so they're a bit more organized